### ISSUE TITLE
Interesting project

### ISSUE THREAD

**Issue by a-fitz-a** [jterrace/js.js#14]
Hi and apologies if this is not the correct place to ask this. This is a really interesting project and would be useful in something I am working on. I am surprised it does not appear to be an active project - is there a reason it has not been developed further? Has anyone implemented it in their software?

Many Thanks

**Comment by jterrace** [comment]
Yeah, it was mainly developed as a proof of concept. It was very difficult to get it to compile, so trying to keep it maintained would be very difficult.
